docs/input/docs/learn/branching-strategies/gitflow/examples.md

```diff
@@ -6,61 +6,79 @@ RedirectFrom:
 - docs/git-branching-strategies/gitflow-examples
 ---
 
-These examples are using the _default_ configuration with GitVersion. Which is
-[continuous deployment](/docs/reference/modes/continuous-deployment) mode for
-`develop` and [continuous delivery](/docs/reference/modes/continuous-delivery) mode
-for all other branches.
-
-This default configuration allows you to publish CI builds from develop to a CI
-MyGet feed, or another CI feed. Then all other branches are manually released
-then tagged. Read more about this at [version increments](/docs/reference/version-increments).
+These examples are illustrating the usage of the supported `GitFlow` workflow in GitVersion. 
+To enable this workflow, the build-in template [GitFlow/v1](/docs/workflows/GitFlow/v1.json) needs to be referenced in the configuration as following:
+```yaml
+workflow: GitFlow/v1
+mode: ContinuousDelivery
+```
+
+Where 
+the [continuous deployment](/docs/reference/modes/continuous-deployment) mode for no branches, 
+the [continuous delivery](/docs/reference/modes/continuous-delivery) mode for `main`, `support` and `develop` branches and 
+the [manual deployment](/docs/reference/modes/manual-deployment) mode for `release`, `feature`, `hotfix` and `unknown` branches 
+are specified.
```

Comment:
This section becomes easier to parse when the links are changed into footnote references.

```suggestion
Where the [continuous deployment][continuous-deployment] mode for no
branches, the [continuous delivery][continuous-delivery] mode for `main`,
`support` and `develop` branches and the [manual
deployment][manual-deployment] mode for `release`, `feature`, `hotfix` and
`unknown` branches are specified.
```